string robot_object
int16 tag_id